sql/codegen.go

```diff
@@ -154,9 +162,12 @@ import logging
 tf.get_logger().setLevel(logging.ERROR)
 ` +
 	// TODO(tonyyang-svail): remove hard coded BATCHSIZE, STEP
+	// TODO(typhoonzero): get NUM_BUCKETS, EMBEDDING_WIDTH from Extended SQL statements
```

Comment:
For every TODO, would it be better to create an issue and we can track it without going through the code?